network/README.md

```diff
@@ -9,6 +9,34 @@ npm install @capacitor/network
 npx cap sync
 ```
 
+## Permissions
+
+On Android, you must add the `ACCESS_NETWORK_STATE` permission to your app's
+`AndroidManifest.xml` file.
+
+```xml
+<uses-permission android:name="android.permission.ACCESS_NETWORK_STATE" />
+```
+
+This permission allows the app to access information about the current network,
+such as whether it is connected to Wi-Fi or cellular.
+
```

Comment:
Same as for haptics, the permission is already added by the plugin https://github.com/ionic-team/capacitor-plugins/blob/main/network/android/src/main/AndroidManifest.xml#L3

so I think we should remove this section, or remove the permission from the manifest